src/Ledger/Deleg.lagda

```diff
@@ -4,64 +4,166 @@
 {-# OPTIONS --safe #-}
 
 open import Ledger.Prelude
+open import Ledger.Epoch
 
-module Ledger.Deleg (Network KeyHash ScriptHash : Set)
-                    ⦃ _ : DecEq Network ⦄ ⦃ _ : DecEq KeyHash ⦄ ⦃ _ : DecEq ScriptHash ⦄ where
+module Ledger.Deleg
+  (Network KeyHash ScriptHash : Set)
```

Comment:
Now that I look at this, I think it's better to require a `crypto : Crypto` here. That includes the hash types with their decidable equality. That also applies to `Ledger.Address`, but to do that there we should also adjust the LaTeX...